extensions/chinese-dictionary/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "chinese-dictionary",
+  "title": "Chinese Dictionary",
+  "description": "Chinese Dictionary",
```

Comment:
How about changing the name to `Mandarin Chinese Dictionary`? It's more accurate. You can also catch users who use `mandarin` as a search keyword.

Reply:
That’s a great idea!

Thanks!